packages/sensors_plus/sensors_plus/android/src/main/kotlin/dev/fluttercommunity/plus/sensors/SensorsPlugin.kt

```diff
@@ -73,9 +73,12 @@ class SensorsPlugin : FlutterPlugin {
     }
 
     companion object {
-        private const val ACCELEROMETER_CHANNEL_NAME = "dev.fluttercommunity.plus/sensors/accelerometer"
+        private const val ACCELEROMETER_CHANNEL_NAME =
+            "dev.fluttercommunity.plus/sensors/accelerometer"
         private const val GYROSCOPE_CHANNEL_NAME = "dev.fluttercommunity.plus/sensors/gyroscope"
-        private const val USER_ACCELEROMETER_CHANNEL_NAME = "dev.fluttercommunity.plus/sensors/user_accel"
-        private const val MAGNETOMETER_CHANNEL_NAME = "dev.fluttercommunity.plus/sensors/magnetometer"
+        private const val USER_ACCELEROMETER_CHANNEL_NAME =
+            "dev.fluttercommunity.plus/sensors/user_accel"
+        private const val MAGNETOMETER_CHANNEL_NAME =
+            "dev.fluttercommunity.plus/sensors/magnetometer"
```

Comment:
since ""dev.fluttercommunity.plus/sensors/" is common in all channel_name, we can use it as a separate const value and append the sensor type later.
something like
```
private const prefix = "dev.fluttercommunity.plus/sensors/'
private const val ACCELEROMETER_CHANNEL_NAME = $prefix + "accelerometer"
```